editoast/src/generated_data/error/track_sections.rs

```diff
@@ -27,14 +27,12 @@ pub fn insert_errors(
     Ok(())
 }
 
-pub fn generate_errors(infra_cache: &InfraCache, graph: &Graph) -> Vec<InfraError> {
-    let mut errors = vec![];
-    errors.extend(check_track_sections(infra_cache, graph));
-    errors
-}
-
 /// Check if routes or buffer stops are missing in the track section
-pub fn check_track_sections(infra_cache: &InfraCache, graph: &Graph) -> Vec<InfraError> {
+pub fn check_track_sections(
+    _: &ObjectCache,
+    infra_cache: &InfraCache,
+    graph: &Graph,
+) -> Vec<InfraError> {
```

Comment:
We can have two function one that check if there is a missing route another one if there is a missing buffer stop.

This is currently really wrong, for each track we call this function that does two checks for **all tracks**. This results in a lot of duplicate errors.